README.Rmd

```diff
@@ -27,31 +27,29 @@ The purpose of this package is to simplify and automate the process of creating
 
 ## **Package context within the R ecosystem**
 
-The package will likely build on the [infer package](https://cran.r-project.org/web/packages/infer/index.html) or the [boot package](https://cran.r-project.org/web/packages/boot/), which allows one to conduct the boostrap sampling in the first place using the generate() and boot() functions. strapR will streamline and extend this process from the pure statistical process done in this module. The [BootCI](https://rdrr.io/cran/DescTools/man/BootCI.html) package seems to streamline the confidence interval process but does not include plotting support. While we cannot be certain that one does not exist, there does not seem to be a package which streamlines the process from data to visualization and presentation as proposed in this document. Some tutorials on bootstrap confidence intervals at <https://moderndive.com/8-confidence-intervals.html>, <http://pages.stat.wisc.edu/~larget/stat302/chap3.pdf> and <https://www.geeksforgeeks.org/bootstrap-confidence-interval-with-r-programming/> encourage the reader to plot the results manually.
+The package will likely build on the [infer package](https://cran.r-project.org/web/packages/infer/index.html) or the [boot package](https://cran.r-project.org/web/packages/boot/), which allows one to conduct the boostrap sampling in the first place using the generate() and boot() functions. strapr will streamline and extend this process from the pure statistical process done in this module. The [BootCI](https://rdrr.io/cran/DescTools/man/BootCI.html) package seems to streamline the confidence interval process but does not include plotting support. While we cannot be certain that one does not exist, there does not seem to be a package which streamlines the process from data to visualization and presentation as proposed in this document. Some tutorials on bootstrap confidence intervals at <https://moderndive.com/8-confidence-intervals.html>, <http://pages.stat.wisc.edu/~larget/stat302/chap3.pdf> and <https://www.geeksforgeeks.org/bootstrap-confidence-interval-with-r-programming/> encourage the reader to plot the results manually.
 
 ## Installation
 
-You can install the released version of strapR from [CRAN](https://CRAN.R-project.org) with:
+You can install the released version of strapr from [CRAN](https://CRAN.R-project.org) with:
 
 ``` r
-install.packages("strapR")
+install.packages("strapr")
 ```
 
 And the development version from [GitHub](https://github.com/) with:
 
 ``` r
 # install.packages("devtools")
-devtools::install_github("UBC-MDS/strapR")
+devtools::install_github("UBC-MDS/strapr")
 ```
 
 ## Example
 
 This is a basic example which shows you how to solve a common problem:
 
-```{r example}
-# library("strapR")
-# density_ci_plot([1, 1, 2, 3, 5, 10]， level=0.95）
-```
+    library("strapr")
```

Comment:
this was the part that was giving me issues when trying to render the readme.md... hopefully this will be resolved when we finish our functions this week